core/sql/regress/tools/runregr_udr.ksh

```diff
@@ -851,9 +851,9 @@ if [ $LINUX -ne 0 ]; then
 fi
 
```

Comment:
Do we want to clean up the unused  env variables/conditional checks ( _RLD_LIB_PATH and CLASSPATH) ?